### PR TITLE
Support for Kafka 2.6 and some scripts and logging fixes

### PR DESCRIPTION
**Pull request by mstruk**

### CODE REVIEW

File: .travis/build.sh [hunk 1]
```diff
@@ -32,7 +32,8 @@ if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
   docker pull oryd/hydra:v1.0.0
   mvn test-compile spotbugs:check -e -V -B -f testsuite
   set +e
-  mvn -e -V -B clean install -f testsuite
+  mvn -e -V -B clean install -f testsuite -Pkafka-2_6_0
+  mvn -e -V -B test -f testsuite -Pkafka-2_5_0
   mvn -e -V -B test -f testsuite -Pkafka-2_4_1
   mvn -e -V -B test -f testsuite -Pkafka-2_4_0
```
Comment (ppatierno) [inline .travis/build.sh:38]: we removed these from the operator. should we still support them here?

File: testsuite/README.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ There are several profiles available to test with a specific version of Kafka im
 - kafka-2_4_0
 - kafka-2_4_1
```
Comment (ppatierno) [inline testsuite/README.md:50]: same questions as above.

File: .travis/build.sh (hunk 1 above)
Comment (mstruk) [reply .travis/build.sh:38]: It does consume a bit more resources when running CI, but for the time being I see no reason to not keep supporting the older versions. People may also be using strimzi-kafka-oauth independently of strimzi-kafka-operator for their own cloud or stand-alone deployments. It's good to detect if something we do breaks compatibility with older versions so we can consider it and mention it in release notes once it happens.

Comment (ppatierno) [reply .travis/build.sh:38]: It makes sense, thanks.

**Pull request merged**